modules/nf-core/biscuit/epiread/main.nf

```diff
@@ -30,8 +30,9 @@ process BISCUIT_EPIREAD {
     def unzipped_snp_bed = snp_bed ? snp_bed.toString() - ~/\.gz$/: ""
     // SNP BED input is optional
     def options_snp_bed = snp_bed ? "-B ${unzipped_snp_bed}" : ""
-    if ("$options_snp_bed" == "${prefix}.bed.gz") error "Input and output names are the same, set prefix in module configuration to disambiguate!"
+    if ("$options_snp_bed" == "${prefix}.bed.gz") error "Input and output names for biscuit epiread are the same, set prefix in module configuration to disambiguate!"
     """
+    set -o pipefail
```

Comment:
nf-core pipelines already have this, cf https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/nextflow.config#L175